creators_custom/RoutesCreatorFenix.py

```diff
@@ -0,0 +1,508 @@
+# coding=utf-8
+
+import os
+import sys
+import pickle
+from collections import OrderedDict
+from datetime import timedelta, datetime
+import overpy
+from osmapi import OsmApi
+from creators_default.RoutesCreator import RoutesCreator
+import creators_custom
+
+CHECK_FOR_NON_PLATFORMS = False
+
+class RoutesCreatorFenix(RoutesCreator):
+
+    def __init__(self, config):
+        self.config = config
```

Comment:
I think it would be better to call the super constructor here which already does assign the config.